src/js/components/ErrorsAlert.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import Alert from './Alert';
+import {getUnanchoredErrorMessage} from '../utils/ErrorMessageUtil';
+
+const ErrorsAlert = function (props) {
+  const {errors, showAllErrors, pathMapping} = props;
+  let showErrors = errors;
+
+  if (!showAllErrors) {
+    showErrors = showErrors.filter(function (error) {
+      return error.path.length === 0;
```

Comment:
Perhaps we could get a comment on the `propType` at the bottom to tell us what `showAllErrors` does. Is it just showing high level errors?